src/main/resources/db/migration/V1_4__feilede_kafka_meldinger.sql

```diff
@@ -0,0 +1,4 @@
+CREATE TABLE FEILEDE_KAFKA_BRUKERE (
+  FODSELSNR VARCHAR2(33) NOT NULL,
```

Comment:
Hvorfor er akkurat 33 valgt som en passende lengde for å holde på det 11-sifrede fødselsnummer? :D Er det dette som brukes i den opprinnelige oppfolgingsbruker-tabellen?
Her hadde det forresten antagelig vært bedre å bruke aktør-id eller Arena sin personid som nøkkel, men den sistnevnte finnes kanskje ikke i User-klassen? Når vi bruker fnr kan vi i teorien oppleve at en bruker feiler og så endrer fødselsnummer før vi får ryddet opp. Dersom dette skjer, vil vi ikke finne personen igjen i oppfolgingsbruker-tabellen, og da vil vel raden bli liggende som feilet til evig tid. Men ser at aktørid også vil bli litt vanskelig, siden denne også kan endre seg, og dessuten ikke finnes i oppfolgingsbruker-tabellen.

Reply:
Hvordan vil vi forresten håndtere brukere som blir borte fra Arena-tabellen? Dette vil vi vel ikke oppdage uten videre. Dette vil iallfall gjelde personer som blir PARBS/RARBS og ikke har "motta selvregistrert person"-oppgaven i Arena. Vi bør kanskje tenkte over om dette kan ha noen konsekvenser.

Reply:
33 er for å ha det samme som arena-tabellen var, egentlig unødvendig, men det var gjort 2 steder fra før av.